### PR TITLE
Add BSIP 45 - "Introduce 'allow use as bitasset backing collateral' flag/permission to assets"

### PR DESCRIPTION
**Pull request by grctest**
```
BSIP: 45
Title: Introduce 'allow use as bitasset backing collateral' flag/permission to assets
Authors: Customminer
Status: Draft
Type: Protocol
Created: N/A
Discussion:  https://github.com/bitshares/bsips/issues/80
Replaces: N/A
Superseded-By: N/A
Worker: N/A
```

### CODE REVIEW

File: bsip-0045.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Proposed UI changes
+
+This flag could only work as long as no MPA has already selected the asset as its backing collateral; setting this as default disabled (not allowed) for newly created assets in the asset creation form could help prevent the asset being used as backing collateral without the user's knowledge.
+
+---
+
+# Summary for Shareholders
+
+* Would likely require a hard fork.
```
Comment (pmconrad) [inline bsip-0045.md]: Please remove that line, or at least the "likely".

**Comment by abitmore** [comment]
Good for me as well.

BTW IIRC someone raised concerns somewhere about the wording of licensing.

**Comment by pmconrad** [comment]
Hm, yes, that's a bit self-contradictory. Also not in line with BSIP-0001:
> Each BSIP must either be explicitly labelled as placed in the public domain (see this BSIP as an example) or licensed under the Open Publication License.

MIT is not an option.

**Comment by grctest** [comment]
BSIPs 21, 22, 23 and 24 also have a similar MIT license line. I don't really care about the licensing of ideas, is the following more appealing? "This document is placed in the public domain." I'll make the change to the 5 docs & update the PR.

**Pull request merged**